eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -187,6 +187,7 @@ jobs:
   - job: 'RunRegression'
     condition: and(succeededOrFailed(), or(eq(variables['Run.Regression'], 'true'), and(eq(variables['Build.Reason'], 'Schedule'), eq(variables['System.TeamProject'],'internal'))))
     displayName: 'Run Regression'
+    timeoutInMinutes: 90
```

Comment:
Do we have an issue tracking the other fix to try and run these in parallel jobs? Also do we thing this time is going to continue to grow or is it pretty stable?

Reply:
https://github.com/Azure/azure-sdk-for-python/issues/11069 <-- that one. It was slightly misnamed (for the original issue needing to increase timeout).